packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, IStyle, ITextFieldStyles, mergeStyles, TextField, useTheme } from '@fluentui/react';
+import React, { useState } from 'react';
+import { useLocale } from '../../localization';
+import { containerStyles, digitStyles, rowStyles, subStyles, textFieldStyles } from './Dialpad.styles';
+
+/**
+ * Strings of {@link Dialpad} that can be overridden.
+ *
+ * @public
+ */
+export interface DialpadStrings {
+  errorText: string;
+  defaultText: string;
+}
+
+/**
+ * Props for {@link Dialpad} component.
+ *
+ * @public
+ */
+export interface DialpadProps {
+  strings?: DialpadStrings;
+  containerStyles?: IStyle;
+  rowStyles?: IStyle;
+  textFieldStyles?: Partial<ITextFieldStyles>;
+  digitStyles?: IStyle;
+  subStyles?: IStyle;
+}
+
+/**
+ * A component to allow users to enter phone number through clicking on dialpad/using keyboard
+ *
+ *
+ * @public
+ */
+export const Dialpad = (props: DialpadProps): JSX.Element => {
+  const localeStrings = useLocale().strings.dialpad;
+  const strings = { ...localeStrings, ...props.strings };
+  return <DialpadContainer errorText={strings.errorText} defaultText={strings.defaultText} {...props} />;
+};
+
+const formatPhoneNumber = (value: string, defaultText: string): string => {
+  // if input value is default text then just return
+  if (value === defaultText) {
```

Comment:
should be able to remove `defaultText` and use placeholder property of the textfield instead

Reply:
updated